src/elements/date-input/date-input.component.ts

```diff
@@ -54,12 +54,9 @@ class SbbDateInputElement<T = Date> extends SbbFormAssociatedInputMixin(LitEleme
     return super.value ?? '';
   }
 
-  @forceType()
-  @property({ attribute: false, type: DateOnlyType })
+  @property({ attribute: false })
```

Comment:
Not sure whether you already told me :D, but why this change?

Reply:
It is no longer necessary, as the setter handles most of the logic anyway.